index.js

```diff
@@ -40,6 +43,18 @@ const responseTime = require('response-time');
 const bodyParser = require('body-parser');
 const randomstring = require('randomstring');
 
+var controllers = {};
+controllers.substitute = require('./controllers/substitute');
+controllers.substitutions = require('./controllers/substitutions');
+
+// code which is executed on every request
+app.use(function(req, res, next) {
```

Comment:
This should not be necessary... can you please remove the CORS headers and see if the tests are still running? Thanks!

Reply:
tests are still running. Deleted this section.